setup.md

```diff
@@ -0,0 +1,334 @@
+# Main setup instructions for the Oxford Computational Biochemistry course
+
+The Oxford Computational Biochemistry course contains 4 self-contained
+tutorials which each have different software installation requirements. These
+instructions can be found in the respective tutorial's folder. Here we
+provide some basic instructions on the base setup requirements common to these
+tutorials and how to obtain a local copy of the materials.
+
+In order to ensure that you have everything ready in advance of the workshop,
+we ask that you **follow these setup instructions before attending each tutorial**.
+Q&A sessions will be setup on the two days preceeding the course to help you
+with any issues you might encounter during the setup process.
+
+
+## 1. [Anaconda]
```

Comment:
Is this a link? I didn't know you could do that...

Reply:
We're like a step away from just writing html at this point 🙃 

That being said - next time let's just write html. It'd be much nicer than having to navigate markdown files.